tests/fields/test_enum_field.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
```

Comment:
this isn't needed/recommended unless you deviate utf8 (https://stackoverflow.com/a/14083123/6203472)

I just realized this was present in other test files so I'm currently fixing that